avsomr/avsomrnetrecognizer.h

```diff
@@ -0,0 +1,36 @@
+#ifndef AVS_AVSOMRNETRECOGNIZER_H
+#define AVS_AVSOMRNETRECOGNIZER_H
+
+#include "all.h"
```

Comment:
not needed, done on commandline

Reply:
Without includes, my IDE (QtCreator) does not highlight types and auto complite do not work. How can I be, what will you advise me?

Reply:
all.h doesn't directly get `#include`d anywhere else as far as I know.
Maybe you need to add `-include ${PROJECT_BINARY_DIR}/all.h` to the compiler flags?

Reply:
> Maybe you need to add -include ${PROJECT_BINARY_DIR}/all.h to the compiler flags?   

It only allows you to find the .h file when #include "all.h". But IDE (QtCreator) can’t build a code model because they don’t have information about the types used and therefore the types don’t highlight, auto complite don’t work, you can’t switch to the declaration and implementation of methods, you can’t find the use, etc.   
Why can't you just include types so that you can see what is used and IDE could help develop?

Reply:
Well, it isn't donw anywhere else, yet seems to work (mostly) reg. syntax highlighting, I'm using Qt Creator too BTW